src/generator.cs

```diff
@@ -5854,7 +5988,7 @@ public void Generate (Type type)
 						print ("return Dlfcn.GetSizeF (Libraries.{2}.Handle, \"{1}\");", field_pi.Name, fieldAttr.SymbolName, library_name);
 					} else if (field_pi.PropertyType == typeof (long)){
 						print ("return Dlfcn.GetInt64 (Libraries.{2}.Handle, \"{1}\");", field_pi.Name, fieldAttr.SymbolName, library_name);
-#if !WATCH
+#if HAVE_COREMEDIA && HAVE_AVFOUNDATION
```

Comment:
I think the `&&` is fine since the next code uses both

```
if (field_pi.PropertyType == typeof (CMTime) ||
	field_pi.PropertyType == typeof (AVCaptureWhiteBalanceGains)){
```